users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -122,19 +136,26 @@ def run_nn_args(nn_args, report_args_collection, dev_corpora, report_name="", re
             "phon_history_length": 0,
             "phon_future_length": 0,
         },
-        scorer_info=score_info,
+        scorer_info=None,
         report=Report(
             columns_start=["train_name"],
             columns_end=["lm_scale", "prior_scale", "sub", "del", "ins", "wer"],
         ),
     )
+    ctc_nn_system._set_scorer("hub5e00", score_info_hub5e00)
     ctc_nn_system.crp["hub5e00"].acoustic_model_config.allophones.add_from_lexicon = False
     ctc_nn_system.crp["hub5e00"].acoustic_model_config.allophones.add_all = True
     ctc_nn_system.crp["hub5e00"].acoustic_model_config.allophones.add_from_file = tk.Path(
         "/u/vieting/setups/swb/20230406_feat/dependencies/allophones_blank",
         hash_overwrite="SWB_ALLOPHONE_FILE_WEI_BLANK",
         cached=True,
     )
+    if use_second_dev_set:
+        ctc_nn_system._set_scorer("hub5e01", score_info_hub5e01)
+        ctc_nn_system.crp["hub5e01"].acoustic_model_config = copy.deepcopy(
+            ctc_nn_system.crp["hub5e00"].acoustic_model_config
+        )
```

Comment:
```suggestion
    if hub5e01_exp_list is not None:
        assert isinstance(hub5e01_exp_list, list)
        
        ctc_nn_system_hub5e01 = copy.deepcopy(ctc_nn_system)
        ctc_nn_system_hub5e01._set_scorer("hub5e01", score_info_hub5e01)
        ctc_nn_system_hub5e01.crp["hub5e01"].acoustic_model_config = copy.deepcopy(
            ctc_nn_system.crp["hub5e00"].acoustic_model_config
        )
        
        for exp in ctc_nn_system_hub5e01.returnn_configs:
            if exp not in hub5e01_exp_list:
                ctc_nn_system_hub5e01.returnn_configs.pop(exp)
        ctc_nn_system_hub5e01.run_dev_recog_step(...)
```
Also, move this part below the `run_train_step` call so the models are available to the system.

And to include the results in the report, you would need to merge both reports. Something like
```
report = ctc_nn_system.report
if hub5e01_exp_list is not None:
    for data in report.data:
        data["eval_set"] = "hub5e00"
    
    for data in ctc_nn_system_hub5e01.report.data:
        data["eval_set"] = "hub5e01"
    report = Report.merge_reports([report, ctc_nn_system_hub5e01.report])
```